test-data/unit/check-newsemanal.test

```diff
@@ -1383,9 +1383,15 @@ x: B
 B = List[C]
 class C(B): pass
 
-reveal_type(x)  # N: Revealed type is 'builtins.list[__main__.C]'
-reveal_type(x[0][0])  # N: Revealed type is '__main__.C*'
+reveal_type(x)
+reveal_type(x[0][0])
 [builtins fixtures/list.pyi]
+[out]
+main:3: error: Cannot resolve name "B" (possible cyclic definition)
+main:4: error: Cannot resolve name "B" (possible cyclic definition)
+main:4: error: Cannot resolve name "C" (possible cyclic definition)
+main:7: note: Revealed type is 'Any'
+main:8: note: Revealed type is 'Any'
```

Comment:
It looks like all new failures _in this file_ are because of the problem with nested placeholders. Please add a comment in the existing issue mentioning this, and add links to the issue in these tests.

Reply:
Added new issues about this: #7111